src/views/create-update-recipe.twig

```diff
@@ -297,17 +316,6 @@
                                 </div>
                             </div>
                         </div>
-                        <div class="zrdn-column">
-                            <label for="total_hours" class="zrdn-label">Total time</label>
-                            <div class="zrdn-field zrdn-is-grouped">
-                                <div>
-                                    <input class="zrdn-input zrdn-is-small" type='number' min="0" id="total_hours" name='total_time_hours' value='{{ total_time_hours }}' />hours
-                                </div>
-                                <div>
-                                    <input class="zrdn-input zrdn-is-small" type='number' min="0" id="total_minutes" name='total_time_minutes' value='{{ total_time_minutes }}' />minutes
-                                </div>
-                            </div>
-                        </div>
```

Comment:
We can also removed `total_time` column from `zrdn_recipe_install` function since we're always calculating it.

Reply:
But it will save front end calculation time and speed up page 
anyways as you suggest i will do that

Reply:
I don't mind about the calculation cost but people might have old total time values in there so we'll leave the column stay for now.

Reply:
Done!
Total will calculate when recipe view called means on run time